src/msw/graphicsd2d.cpp

```diff
@@ -5169,7 +5189,13 @@ wxD2DRenderer::~wxD2DRenderer()
 
 wxGraphicsContext* wxD2DRenderer::CreateContext(const wxWindowDC& dc)
 {
-    return new wxD2DContext(this, m_direct2dFactory, dc);
+    wxD2DContext* d2d = new wxD2DContext(this, m_direct2dFactory, dc);
```

Comment:
It would be nice to make a factory function to be able to write just

```cpp
return wxD2DContext::New(this, m_direct2dFactory, dc);
```

here and below instead of repeating the same code in multiple places.